.setup/options/web-develop/docker/run.sh

```diff
@@ -18,7 +18,8 @@ if ! command -v $COMMAND_NAME &>/dev/null; then
 
     echo 'Install docker compose:'
     echo 'Check new version at https://github.com/docker/compose/releases'
-    DOCKER_COMPOSE_VERSION=v2.17.1
+    echo 'Auto checking for latest version...'
+    DOCKER_COMPOSE_VERSION=$(curl -s https://api.github.com/repos/docker/compose/releases/latest | grep 'tag_name' | cut -d '"' -f 4)
```

Comment:
curl is used to get the latest version information from the Docker website. The grep statement is then used to filter out the value of the 'tag_name' field. Finally, the cut statement is used to get the fourth value in a double quote-separated string, which is the latest version of Docker Compose.